controllers/storagecluster/generate.go

```diff
@@ -111,10 +109,6 @@ func generateNameForCephNetworkFilesystemSC(initData *ocsv1.StorageCluster) stri
 	return fmt.Sprintf("%s-ceph-nfs", initData.Name)
 }
 
-func generateNameForNFSCSIProvisioner(initData *ocsv1.StorageCluster) string {
```

Comment:
Why are we removing this func. I see this was being used.

Reply:
IT was only used in one place and that too not consuming the parameter passed to it. So, removed it to directly use the return value of `fmt.Sprintf`.
We are doing a similar thing for a function just above it as well.
Ref: https://github.com/red-hat-storage/ocs-operator/blob/fbfe2d9ea0256d9e38ee57ca911c74169cf55f6d/controllers/storagecluster/storageclasses.go#L377